core/trie/trie.go

```diff
@@ -17,6 +17,8 @@ type Storage interface {
 	Delete(key *bitset.BitSet) error
 }
 
+type HashFunc func(*felt.Felt, *felt.Felt) *felt.Felt
+
```

Comment:
Any particular reason this is public? 

We should only make it public if `NewTrie` is also public. Which is currently not the case.

Reply:
made private